dali/python/nvidia/dali/ops/__init__.py

```diff
@@ -189,6 +189,116 @@ def _handle_deprecations(schema, kwargs, op_name):
     return kwargs
 
 
+def _handle_op_deprecation(schema, op_name):
+    if schema.IsDeprecated():
+        # TODO(klecki): how to know if this is fn or ops?
```

Comment:
So for the class-based API we're going to suggest `fn.*` to now? Was it like that before?

Reply:
Yup, this was just moved around.
I think the best way would be to add some `_api_type` string and dispatch on that in errors (also propagate it to the backend as hidden arg possibly. But this is a case for follow-up.